### PR TITLE
Deeplinking tab app m365

### PR DESCRIPTION
**Pull request by panksahu**
Sample app for deeplinking
- Implemented below scenarios

1. App to App scenarios
2. App to Hub scenarios

### CODE REVIEW

File: deep-linking-hello-world-tab-without-sso-M365/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+>
+> We really appreciate your feedback! If you encounter any issue or error, please report issues to us following the [Supporting Guide](./../SUPPORT.md). Meanwhile you can make [recording](https://aka.ms/teamsfx-record) of your journey with our product, they really make the product better. Thank you!
+>
+> This warning will be removed when the samples are ready for production.
+
+Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.
+
+Hello World Tab shows you how to build a tab app without single sign on.
+
+![Hello World Tab Without SSO](images/hello-world-tab-without-sso.gif)
```
Comment (HuihuiWu-Microsoft) [inline deep-linking-hello-world-tab-without-sso-M365/README.md:13]: You may need to update the README

File: deep-linking-hello-world-tab-without-sso-M365/cli.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Try sample with TeamsFx CLI
+
+1. Install [Node.js](https://nodejs.org/en/download/) (use the latest v14 LTS release)
+1. To install the TeamsFx CLI, use the npm package manager:
+    ```
+    npm install -g @microsoft/teamsfx-cli
+    ```
+1. Create hello-world-tab project.
+    ```
+    teamsfx new template hello-world-tab-without-sso
```
Comment (HuihuiWu-Microsoft) [inline deep-linking-hello-world-tab-without-sso-M365/cli.md:10]: The sample id should be updated

File: deep-linking-hello-world-tab-without-sso-M365/templates/appPackage/manifest.template.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
+{
+    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
```
Comment (HuihuiWu-Microsoft) [inline deep-linking-hello-world-tab-without-sso-M365/templates/appPackage/manifest.template.json:2]: Any reason to use this preview version?

Comment (vatsal121) [reply deep-linking-hello-world-tab-without-sso-M365/templates/appPackage/manifest.template.json]: Fixed.

File: deep-linking-hello-world-tab-without-sso-M365/README.md (hunk 1 above)
Comment (vatsal121) [reply deep-linking-hello-world-tab-without-sso-M365/README.md]: I have updated the Readme

File: deep-linking-hello-world-tab-without-sso-M365/cli.md (hunk 2 above)
Comment (vatsal121) [reply deep-linking-hello-world-tab-without-sso-M365/cli.md]: Sample id is updated

File: deep-linking-hello-world-tab-without-sso-M365/.fx/configs/projectSettings.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        ],
+        "activeResourcePlugins": [
+            "fx-resource-frontend-hosting",
+            "fx-resource-identity",
+            "fx-resource-local-debug",
+            "fx-resource-appstudio"
+        ]
+    },
+    "programmingLanguage": "typescript",
+    "projectId": "888e051f-587a-4e02-ba80-172aa85c9332"
```
Comment (HuihuiWu-Microsoft) [inline deep-linking-hello-world-tab-without-sso-M365/.fx/configs/projectSettings.json:21]: Can you help remove the 'projectId' field(both the key and value)?

Comment (vatsal121) [reply deep-linking-hello-world-tab-without-sso-M365/.fx/configs/projectSettings.json]: I have removed the 'projectId' field.

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

File: deep-linking-hello-world-tab-without-sso-M365/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
+# Getting Started with Hello World Deeplinking Tab Without SSO Sample
+
+> Note: Please be advised that this sample repository is currently in **Public Preview**, with a lot of active development work taking place. Please expect breaking changes as we continue to iterate.
```
Comment (summzhan) [inline deep-linking-hello-world-tab-without-sso-M365/README.md:3]: Note in line3 should be removed, we are GA now.

File: deep-linking-hello-world-tab-without-sso-M365/README.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+>
+> We really appreciate your feedback! If you encounter any issue or error, please report issues to us following the [Supporting Guide](./../SUPPORT.md). Meanwhile you can make [recording](https://aka.ms/teamsfx-record) of your journey with our product, they really make the product better. Thank you!
+>
+> This warning will be removed when the samples are ready for production.
```
Comment (summzhan) [inline deep-linking-hello-world-tab-without-sso-M365/README.md:7]: remove the line 7.

File: deep-linking-hello-world-tab-without-sso-M365/README.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+2. App to Hub Scenarios
+    - Deeplinking from outside app
+    - Navigating from within the hub 
+
+![Hello World Deeplinking Tab](images/deeplink-without-SSO.gif)
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/), fully tested on NodeJS 12, 14
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
```
Comment (summzhan) [inline deep-linking-hello-world-tab-without-sso-M365/README.md:24]: "A Microsoft 365 account". M365 is not the official branding, hence cannot appear in our public doc. We should use the correct branding which is Microsoft 365

File: deep-linking-hello-world-tab-without-sso-M365/README.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+## What you will learn in this sample:
+
+- How to use TeamsFx to build frontend hosting on Azure for your navigation tab app which simplify the process of deeplink navigation across the hubs (on Teams, Outlook and Office).
+
+## Try the Sample with Visual Studio Code Extension:
+>Here are the instructions to run the sample in **Visual Studio Code**. You can also try to run the app using TeamsFx CLI tool, refer to [Try the Sample with TeamsFx CLI](cli.md)
+
+1. Clone the repo to your local workspace or directly download the source code.
+1. Download [Visual Studio Code](https://code.visualstudio.com) and install [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit).
```
Comment (summzhan) [inline deep-linking-hello-world-tab-without-sso-M365/README.md:35]: "... and install [Teams Toolkit Visual Studio Code Extension]":
better to call it "Teams Toolkit for Visual Studio Code"

File: deep-linking-hello-world-tab-without-sso-M365/README.md (hunk 5 above)
Comment (panksahu) [reply deep-linking-hello-world-tab-without-sso-M365/README.md]: Removed & updated.

**Pull request merged**